plat/kvm/Config.uk

```diff
@@ -97,6 +97,9 @@ config KVM_VMM_QEMU
 	bool "QEMU"
 	imply LIBUKCONSOLE_PL011 if ARCH_ARM_64
 	imply LIBUKCONSOLE_PL011_EARLY_CONSOLE if ARCH_ARM_64
+	imply LIBUKCONSOLE_NS16550 if ARCH_X86_64
+	imply LIBUKCONSOLE_NS16550_COM1 if ARCH_X86_64
+	imply LIBUKCONSOLE_NS16550_COM_EARLY_CONSOLE if ARCH_X86_64
```

Comment:
When seeing this, I am thinking that we should use the same config name for the early console boolean for X86 and Arm: `LIBUKCONSOLE_NS16550_COM_EARLY_CONSOLE`:
```
	imply LIBUKCONSOLE_NS16550 if (ARCH_ARM_64 || ARCH_X86_64)
	imply LIBUKCONSOLE_NS16550_EARLY_CONSOLE
	imply LIBUKCONSOLE_NS16550_COM1 if ARCH_X86_64
```